### DISCUSSION TITLE
Using https in devserver preferably mkcert

### DISCUSSION THREAD

**Discussion by pdmshrestha** [storybookjs/storybook#22698] [Help]
Hi, 

I have using [mkcert vite plugin](https://github.com/liuweiGL/vite-plugin-mkcert) for ssl support on vite project. Is there a way to configure storybook devserver to use same certificate from mkcert. Tried following the documentation from [storybook](https://storybook.js.org/docs/vue/builders/vite#configuration) but could not figure it out.

**Comment by IanVS** [comment]
Right now storybook uses an express dev server with Vite in a middleware mode, though I experimented a while back with using fastify instead (https://github.com/storybookjs/storybook/issues/17971).  I'm not positive what the http/2 story is for express right now, it looks like maybe they finally added support?  But you'd need to use the `--https` option and pass cert information to Storybook's cli: https://storybook.js.org/docs/react/api/cli-options.  I'd be curious to hear if you have any improvements by doing so!

**Reply by scarf005** [reply]
i think storybook should respect `https` option in vite.config.ts; for now it ignores the option.
